### PR TITLE
Push catalog info after the checks

### PR DESCRIPTION
**Pull request by dottorblaster**
Right now if the database goes bananas we don't repopulate the table containing the catalog data. This PR attempts to address the issue making that happen after the checks ran.

### CODE REVIEW

**Review by arbulu89 — CHANGES_REQUESTED** [review]
Hey @dottorblaster ,
Thanks! Unfortunately this will add some overload to the checks execution, but well it is what it is. Hopefully in the future we can add some kind of logic to the runner to get commands from the web itself (or we might even remove the runner at all, who knows hehe).

I recommend that once we add the metadata execution in the loop, we could remove the initial execution (line 87), so we only execute meta and checks within the loop. And I would execute the metadata and checks in this order.

File: runner/runner.go [hunk 1]
```diff
@@ -212,10 +217,16 @@ func (c *Runner) startCheckRunnerTicker() {
 		}
 
 		if err = checkRunner.SetInventory(inventoryFile); err != nil {
+			log.Errorf("Error setting the ansible inventory file")
 			return
 		}
 
 		checkRunner.RunPlaybook()
+
+		if err = metaRunner.RunPlaybook(); err != nil {
```
[the remark this answers is not in the document]
Comment (dottorblaster) [reply runner/runner.go]: Agree on the order, I moved it on top!

**Comment by dottorblaster** [comment]
@arbulu89 100% agree about the order of events, about the removal of the first run on boot I honestly don't know, because I prefer to run this also at the boot to ensure the user doesn't see an empty screen for too long at first. What do you think?

**Comment by arbulu89** [comment]
> @arbulu89 100% agree about the order of events, about the removal of the first run on boot I honestly don't know, because I prefer to run this also at the boot to ensure the user doesn't see an empty screen for too long at first. What do you think?

I don't think removing this would change anything. This is the current logic:
```
run_meta()
while true:
   run_meta()
   run_checks()
```

If we remove the first `run_meta()`, we will still have 1 `run_meta` executed in the 1st place. Or am I missing something?

**Comment by dottorblaster** [comment]
@arbulu89 I was assuming that the first tick executed after the first interval, sorry about that :sweat_smile: I just noticed that the first tick runs immediately. I'm going to remove the first call then :smile:

**Pull request merged**